.gitignore

```diff
@@ -1,29 +1,32 @@
-# Intellij
-*.iml
-.idea
-
-# npm
-node_modules
-yarn-error.log
-
-# build
-/main.js
-*.js.map
-.log
-
-# obsidian
-/data.json
-/.obsidian
-
-# osx
-.DS_Store
-
-# asdf
-.tool-versions
-
-# files in sample vaults
-/resources/sample_vaults/Tasks-Demo/.obsidian/workspace
-.hotreload
-
-# Backup files.
-*.bak
+# Intellij
+*.iml
+.idea
+
+# npm
+node_modules
+yarn-error.log
+
+# build
+/main.js
+*.js.map
+.log
+
+# obsidian
+/data.json
+/.obsidian
+
+# jest coverage files
+/coverage
+
+# osx
+.DS_Store
+
+# asdf
+.tool-versions
+
+# files in sample vaults
+/resources/sample_vaults/Tasks-Demo/.obsidian/workspace
+.hotreload
```

Comment:
I think there must be an inconsistency in the line endings here.

It is showing the whole file as having been modified.

Maybe it doesn't matter for this one, but it would be helpful for future PRs to understand why...

Reply:
I'm curious as to why the workspace file needs to be ignored - why this didn't show up before.
I wonder whether there is a .gitignore in a lower level folder that actually ignored it already?